internal/host/validator.go

```diff
@@ -234,6 +235,12 @@ func boolValue(b bool) ValidationStatus {
 	}
 }
 
+func getFunctionName() string {
```

Comment:
IIUC, this is getting the parent's function name, right?

Should we call this `getParentFunctionName`

Reply:
the idea this to print the currently running function name.
for example: the `isConnected()` validation function will print:

`github.com/openshift/assisted-service/internal/host.(*validator).isConnected validation result for host 22bf2c75-1f7b-4ddf-afed-ccd8e041b321 is: success`